client/app/cloud/project/compute/infrastructure/flavor.service.js

```diff
@@ -0,0 +1,177 @@
+(() => {
+
+    const oldFlavorRegex = /eg|sp|hg|vps-ssd/;
+
+    class CloudFlavorService {
+
+        constructor ($filter, CLOUD_FLAVORTYPE_CATEGORIES, CLOUD_INSTANCE_CPU_FREQUENCY) {
+            this.$filter = $filter;
+            this.CLOUD_FLAVORTYPE_CATEGORIES = CLOUD_FLAVORTYPE_CATEGORIES;
+            this.CLOUD_INSTANCE_CPU_FREQUENCY = CLOUD_INSTANCE_CPU_FREQUENCY;
+        }
+
+        static isOldFlavor (flavorName) {
+            return oldFlavorRegex.test(flavorName);
+        }
+
+        static getFlavorTypes (flavors) {
+            return _.uniq(_.pluck(flavors, "type"));
+        }
+
+        static addPriceInfos (flavor, prices) {
+            const price = { price: { value: 0 }, monthlyPrice: { value: 0 } };
+            const planHourly = prices[_.get(flavor, "planCodes.hourly")];
+            if (planHourly) {
+                _.set(price, "price", planHourly.price);
+                // Set 3 digits for hourly price
+                _.set(price, "price.text", _.get(price, "price.text", "").replace(/\d+(?:[.,]\d+)?/, `${price.price.value.toFixed(3)}`));
```

Comment:
Are we sure this regex cover every currencu case?

1 000 000.00
1 000 000,00
1000000.00
1000000,00

I even thing that 
1,000,000.00 is a thing.

So make sure to test all locales ;)